testfm/constants.py

```diff
@@ -1,17 +1,25 @@
-import configparser
+from dynaconf import LazySettings
 
-config = configparser.ConfigParser()
-config.read("testfm.properties")
+settings = LazySettings(
```

Comment:
and then initiate the object as:
```suggestion
settings = Dynaconf(
```